x-pack/plugins/infra/server/lib/alerting/log_threshold/register_log_threshold_rule_type.ts

```diff
@@ -127,6 +131,9 @@ export async function registerLogThresholdRuleType(
           name: 'denominatorConditions',
           description: denominatorConditionsActionVariableDescription,
         },
+        ...(getAlertDetailsPageEnabledForApp(config, 'logs')
+          ? [{ name: 'alertDetailsUrl', description: alertDetailUrlActionVariableDescription }]
+          : []),
```

Comment:
Is it intentional that the feature flag only controls the visibility of the context variable in the dropdown, but doesn't prevent the variable from being injected into the context in the first place?

Reply:
@weltenwort It was not considered harmful by the team to already include the URL in the alert context as the frontend also has a feature flag to block access to the page.

Once the feature flag is set to true, in this approach the user could visit the alert details page for alerts that already fired earlier.